lib/utils/formatter.go

```diff
@@ -326,7 +326,7 @@ func (w *writer) writeMap(m map[string]interface{}) {
 			continue
 		}
 		switch value := m[key].(type) {
-		case log.Fields:
+		case map[string]interface{}:
```

Comment:
Here and maybe in the signature too:

```suggestion
		case map[string]any:
```

Reply:
Done.